redisinsight/api/src/modules/redis/client/redis.client.ts

```diff
@@ -203,6 +203,7 @@ export abstract class RedisClient extends EventEmitter2 {
         server_name: helloResponse.server,
         redis_version: helloResponse.version,
         redis_mode: helloResponse.mode,
+        info_command_is_disabled: true,
```

Comment:
Let's make it a RedisClient property? And update each time right here (set to false or set to true when `info` command respond without error)